zerver/webhooks/teamcity/fixtures/success_branch.json

```diff
@@ -0,0 +1,54 @@
+{
+  "build": {
+    "buildStatus": "Tests passed: 27",
+    "buildResult": "success",
+    "buildResultPrevious": "success",
+    "buildResultDelta": "unchanged",
+    "notifyType": "buildFinished",
+    "buildFullName": "Project :: Compile",
+    "buildName": "Compile",
+    "buildId": "19952",
+    "buildTypeId": "Project_Compile",
+    "buildInternalTypeId": "bt1",
+    "buildExternalTypeId": "Project_Compile",
+    "buildStatusUrl": "http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952",
+    "buildStatusHtml": "<span class=\"tcWebHooksMessage\"><a href=\"http://teamcity/project.html?projectId=Project\">Project</a> :: <a href=\"http://teamcity/viewType.html?buildTypeId=Project_Compile\">Compile</a> # <a href=\"http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952\"><strong>5535 - CL 123456</strong></a> has <strong>finished</strong> with a status of <a href=\"http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952\"> <strong>success</strong></a> and was triggered by <strong>Perforce</strong></span>",
+    "rootUrl": "http://teamcity",
+    "projectName": "Project",
+    "projectId": "Project",
+    "projectInternalId": "project1",
+    "projectExternalId": "Project",
+	"branchDisplayName": "MyBranch",
+    "buildNumber": "5535 - CL 123456",
+    "agentName": "agent123",
+    "agentOs": "Windows 10, version 10.0",
+    "agentHostname": "agent123.company.com",
+    "triggeredBy": "Perforce",
+    "message": "Build Project :: Compile has finished. This is build number 5535 - CL 123456, has a status of \"success\" and was triggered by Perforce",
+    "text": "Project :: Compile has finished. Status: success",
+    "buildStateDescription": "finished",
+    "buildRunners": [
+      "Command Line",
+      "Command Line",
+      "Command Line",
+      "Command Line",
+      "Command Line",
+      "Command Line"
+    ],
+    "extraParameters": [ ],
+    "teamcityProperties": [
+      {
+        "name": "DotNetFramework2.0.50727_x64_Path",
+        "value": "C:\\Windows\\Microsoft.NET\\Framework64\\v2.0.50727"
+      },
+      {
+        "name": "DotNetFramework2.0.50727_x86_Path",
+        "value": "C:\\Windows\\Microsoft.NET\\Framework\\v2.0.50727"
+      },
+      {
+        "name": "DotNetFramework2.0_x64",
+        "value": "2.0.50727"
+      }
+    ]
+  }
+}

```

Comment:
Could you please format this with 4-space indentation? It is easier on the eyes. :) See [the end of this document](https://zulip.readthedocs.io/en/latest/tutorials/documenting-api-endpoints.html#formatting-json) for a quick way to do that! :)

Reply:
done =)